app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1441,3 +1441,32 @@ $include-html-global-classes: $include-html-classes;
 // $include-accessibility-classes: true;
 
 @import 'foundation';
+
+.dash-header {
+  background-color: $topbar-bg-color;
+}
+
+.footer {
+  background-color: $primary-color;
+  text-align: center;
+  color: #fff;
+
+  i {
+  font-size: 2rem;
+  }
+
+  p, a {
+  font-size: .8em;
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>`.8` should be written with a leading zero as `0.8`<br>Properties should be ordered color, font-size, margin-top